apis/openstack-nova/src/main/java/org/jclouds/openstack/nova/v2_0/compute/functions/ImageToOperatingSystem.java

```diff
@@ -80,9 +80,9 @@ public OperatingSystem apply(final Image from) {
             is64Bit = matcher.group(2).equals("x64");
          }
       } else {
-         if (imageName.contains("Red Hat EL")) {
+         if (imageName.contains("Red Hat")) {
```

Comment:
Well, the imageName string changed from "EL" to "Enterprise Linux" at some point, which is why the match was failing, but I don't know the original reasoning for being that specific. Seems safe, though.